services/job_chat/evaluation/test_prompts_issue_97_a.ipynb

```diff
@@ -0,0 +1,966 @@
+{
```

Comment:
Can I ask you to rename both notebooks to something more meanigful? issue 97 is too broad.

Maybe like:
```
test_nested_state
test_related_questions
````